### ISSUE TITLE
Memory leak when changing shaders normal map, etc

### ISSUE THREAD

**Issue by ghost** [asiekierka/FoamFix#252]
Memory leak and almost endless graphic changing with your mod

**Comment by asiekierka** [comment]
Yes, I'm aware - this bug has been in since nearly the beginning. Unfortunately, the mod is not currently maintained, so this issue will have to wait.

**Comment by hawei1213** [comment]
on what shader this occurs?
